tests/testthat/setup.R

```diff
@@ -0,0 +1,79 @@
+
+## SET UP ----
+
+  # Set teardown environment
+  teardownEnv <- if (testthat::is_testing()) testthat::teardown_env() else .GlobalEnv
+
+  # Set module path
```

Comment:
Is there a better way to set the module path?

Reply:
TBD when module is temporarily converted to a package with a temporary DESCRIPTION file.